### DISCUSSION TITLE
Week 11 - Progress Update

### DISCUSSION THREAD

**Discussion by coopsand** [USU-ITLS-x265-23-summer/class-repo#240] [General]
Hello everyone! Here is the status of my project:
<img width="867" alt="Screenshot 2023-08-06 at 12 39 45 PM" src="https://github.com/USU-ITLS-x265-23-summer/class-repo/assets/133150498/4abace35-7776-40e8-80f6-76d846f4f83a">

All the content is here! I have worked really hard to create a contact form from memory - based upon what I remember from the FreeCodeCamp exercise. I also added descriptions to the Graphic Design pieces, which was missing last week.

I want to now just tweak the CSS for the project. I want to change how a couple things look and feel. I also want to see if I can add a little bit of JS for the form as well. 

Hope you all are feeling satisfied with your projects!!

**Comment by coopsand** [comment]
Comments:
- https://github.com/USU-ITLS-x265-23-summer/bst3ck-final-project/commit/8c0800329b79d655849b9adff53b8f4941cee6d6
- https://github.com/USU-ITLS-x265-23-summer/Skye404Project/commit/94bbb6a396546251f69a662d24228984ca19de47

**Comment by kassyoveson** [comment]
Great job! I just looked at your page. The pages flowed well. I like how the links actually worked:) We are in the same major, so seeing some of your projects has inspired me what to add to my portfolio. My question is why did you chose the color red for your fonts? I am just curious on why you chose that color.

**Reply by coopsand** [reply]
That's a good question. There were a couple of reasons that I chose red. First, it's one of my favorite colors, and I feel that it is very bold and eye-catching. Secondly, I am largely focusing on accessibility in my site, so by choosing a dark red, it would be very easy to contrast against a lighter background to comply with WCAG AA standards.
